### PR TITLE
Several changes:

### PR DESCRIPTION
**Pull request by raticate**
1. connected to the node database with an environment variable
2. obtained node public key from node configuration file. The path to the configuration file is defined in an environmental variable.
3. Now, instead of being a python module, the app is a python package.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class DefaultConfig(object):
     DEBUG = True
     CSRF_ENABLED = True
-    SECRET_KEY = '[\xd7\xda\x0cZ\x8a8\x8b\xb1g~\x0b\x86\x96\xf5\xe3\xa5Q\xfb\x86\x03\xb2\xfc'
-    SQLALCHEMY_DATABASE_URI = 'sqlite:////home/caterina/node_1.db'
-    SQLALCHEMY_TRACK_MODIFICATIONS = False
\ No newline at end of file
+    SECRET_KEY = 'app secret key'
```
Comment (fmontoto) [inline config.py:4]: I hope you generated a different one, as the deleted one will stay here forever.

File: runserver.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from nodeadmin import app
+app.run(debug=True)
```
Comment (fmontoto) [inline runserver.py]: No not run with debug=True as default

File: config.py (hunk 1 above)
Comment (raticate) [reply config.py:4]: I know, I already generated a new one

**Comment by fmontoto** [comment]
There is a .pyc file, remove it and append *.pyc to .gitignore

File: runserver.py (hunk 2 above)
Comment (fmontoto) [reply runserver.py]: Do not*

**Pull request merged**